src/proto/gps.proto

```diff
@@ -20,6 +20,9 @@ enum SampleType {
   TOTAL_DATA_TYPES = 13;
   POSITION = 14;
   LINEAR_VELOCITY = 15;
+  RGB_IMAGE_SIZE = 16;
+  DOMAIN_IMAGE = 17;
```

Comment:
Hi,
It captures only the first image (at t=0) and sets that as the observation
for all time steps.
I'm using it as the 'image of the domain', hence the name.

Aviv

On Mon, Apr 18, 2016 at 11:50 AM, Bradly Stadie <notifications@github.com>
wrote:

> In src/proto/gps.proto
> <https://github.com/cbfinn/gps/pull/15#discussion_r60113292>:
>
> > @@ -20,6 +20,9 @@ enum SampleType {
> >    TOTAL_DATA_TYPES = 13;
> >    POSITION = 14;
> >    LINEAR_VELOCITY = 15;
> > +  RGB_IMAGE_SIZE = 16;
> > +  DOMAIN_IMAGE = 17;
>
> @avivt <https://github.com/avivt> Can you shed any light on what
> DOMAIN_IMAGE is doing? I couldn't tell by reading the code.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cbfinn/gps/pull/15/files/7cb072155dd03779a3cdcaca47d1c59ef5ba1449#r60113292>
>